### PR TITLE
WIP Add Dockerfile for Fedora

### PR DESCRIPTION
**Pull request by xliiv**
Issue related #2126

### CODE REVIEW

File: run.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```
Comment (xliiv) [inline run.sh]: This file will be removed

**Comment by xliiv** [comment]
`Cargo rpm build` gives an error

```
   Compiling alacritty v0.3.2 (/source/alacritty)
   Compiling nom v4.2.3
   Compiling url v1.7.2
   Compiling wayland-scanner v0.21.12
   Compiling wayland-scanner v0.22.2
error: failed to run custom build command for `alacritty v0.3.2 (/source/alacritty)`
process didn't exit successfully: `/source/target/release/build/alacritty-dcce14aa21e0720c/build-script-build` (exit code: 101)
--- stderr
thread 'main' panicked at 'couldn't get commit hash', src/libcore/option.rs:1038:5
note: Run with `RUST_BACKTRACE=1` environment variable to display a backtrace.

warning: build failed, waiting for other jobs to finish...
error: build failed
```
debugging it

**Comment by chrisduerr** [comment]
@xliiv See https://github.com/jwilm/alacritty/pull/2443.

**Comment by xliiv** [comment]
> @xliiv See #2443.

@chrisduerr , should i wait until it gets merged?

Thanks for helping with this.

**Comment by chrisduerr** [comment]
> should i wait until it gets merged?

That's probably a good idea, though feel free to get it working by basing it on top of the PR if you'd like to. However merging it will likely only make sense once #2443 is resolved.

**Comment by xliiv** [comment]
@chrisduerr who can i ask for a review?

**Comment by chrisduerr** [comment]
@xliiv I'll check this out once the git PR is merged. If it works I'll rebase myself and merge it as is, otherwise I'll let you know if there's anything else left to be done.

**Comment by xliiv** [comment]
@chrisduerr thanks!

**Comment by chrisduerr** [comment]
Okay, so after a bit of discussion in https://github.com/jwilm/alacritty/pull/2506, I think that is likely the better way to solve this Fedora installation problem.

Hopefully that should provide a nice way for everyone to just install Alacritty on all Fedora systems without us actually having to build inside a specific docker container.

Thanks again for the PR and sorry it didn't work out this time.

**Comment by xliiv** [comment]
No problem, if we get better experience, i'm happy with that. Thanks!

**Pull request closed without merging**